### DISCUSSION TITLE
gatsby initial Routing doesn't work on Production

### DISCUSSION THREAD

**Discussion by levani-k** [gatsbyjs/gatsby#31443] [Help]
Problem:
when i visit "https://wenroll.com/about" it redirects back to the homepage only on production. is there any more information on gatsby routing? cuz i cannot found anything in doc which would explain this behavior.

Seems good:
if I write this "https://wenroll.com/about" in url it redirects me to home page. if I visit about page by clicking button in header it redirects me to about page.

Spent a whole day on this and got nowhere!

**Comment by LekoArts** [comment]
If this only happens on production and not locally after a `gatsby build` and `gatsby serve` you'll need to check your hosting and/or provide more information for people to help you :)
